brianlong/www.validators.app
# 186256273 geofence ips by country

#### What's this PR do?
- determine the country of origin of the user
- restrict staking links on validators detail page for a specific country

#### How should this be manually tested?
- run tests
- uncomment last line in set_geo_country method and try various IPs
- check if links are displayed according to condition

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186256273)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [Y] Does this PR add new dependencies?
